dvc/repo/__init__.py

```diff
@@ -185,10 +185,22 @@ def find_dvc_dir(cls, root=None):
         return os.path.join(root_dir, cls.DVC_DIR)
 
     @staticmethod
-    def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
+    def init(
+        root_dir=os.curdir,
+        no_scm=False,
+        force=False,
+        subdir=False,
+        ignore_template_list=None,
+    ):
         from dvc.repo.init import init
 
-        init(root_dir=root_dir, no_scm=no_scm, force=force, subdir=subdir)
+        init(
+            root_dir=root_dir,
+            no_scm=no_scm,
+            force=force,
+            subdir=subdir,
+            ignore_template_list=ignore_template_list,
```

Comment:
People will probably want these even in existing projects, and they will probably forget to do this during the init phase. So we could add another command to install these, but maybe it is actually better to just document that people might want to get .dvcignore from https://github.com/github/gitignore  and use it.

This also makes me wonder if we should should really somehow reuse existing .gitignores (e.g. to not search for dvc-files and .dvcignores in .gitignore'd directories). But I suppose many people also want to exclude these dirs from dirs they are `dvc add`ing (or tracking), which doesn't play well, because we gitignore the data we are adding. :thinking:

Reply:
@efiop, Ok. May be we should add command for downloading template content from https://github.com/github/gitignore and pasting it in ```.dvcignore```? Then, we need not to store it in dvc code base and mantain, and user need not to do it by hand. 

And also we can add second option for this command to read existing ```,gitignore```, exlude all records that was added by ```dvc add``` and paste it in ```.dvcignore```.  Is it possible to easily get a list of all tracked files? (I think it should be, I just don't know where to look).

What do you think of these two possibilities?

Reply:
> @efiop, Ok. May be we should add command for downloading template content from https://github.com/github/gitignore and pasting it in .dvcignore? Then, we need not to store it in dvc code base and maintain, and user need not to do it by hand.

@aerubanov I don't think it is worth creating a special command. It is not hard to just go to a link in the docs and copypaste contents to .dvcignore (or use wget/curl/etc).

> And also we can add second option for this command to read existing ,gitignore, exlude all records that was added by dvc add and paste it in .dvcignore. Is it possible to easily get a list of all tracked files? (I think it should be, I just don't know where to look).

That sounds a bit complex :( We had a discussion with @pmrowla about it, and maybe we could just use `.gitignore` directly, the only thing that is problematic is not dvcignoring dvc-tracked files, but that should be solvable by telling dvcignore about them  when we collect the dag. We will soon have a pre-requisite PR that notifices dvcignore about those outputs for performace reasons and we might get back to this idea right after it.

Reply:
Ok, i will make the necessary changes.